src/pages/SendManyTransferDetails.tsx

```diff
@@ -12,9 +22,9 @@ export default function SendManyTransferDetails({ userSession, txId, eventIndex
         </div>
         <div className="col-xs-10 col-md-8 mx-auto mb-4 px-4">
           <SendManyTransfer
-            txId={txId}
-            eventIndex={eventIndex}
-            ownerStxAddress={ownerStxAddress}
+            txId={txId!}
```

Comment:
Why are these properties optional arguments when you enforce them later?

Reply:
this is due to @reach/router not properly typing the parameters so I have to use this funky arrangement in regular js it wouldn't complain